### PR TITLE
Fixes  #918 - Clear table problems link on monitor page do nothing.

### LINKED ISSUE #918: Clear table problems link on monitor page do nothing.
While running continuous ingest against 2.0.0-alpha-2 I saw the following.   Clicking on the links to clear the problems did nothing.

screenshot at 2019-01-24 17 32 28

### PR DESCRIPTION
**Pull request by jzgithub1**
The Regexes in the validators in the ProblemsResource REST object had mistakes and a new regex neeed to be added.

### CODE REVIEW

**Comment by mjwall** [comment]
@jzgithub1 you have a checkstyle violation.  Follow through the details of the travis-ci build or go directly to https://travis-ci.org/apache/accumulo/jobs/493247193.

Consider rebase -i to clean up the commits and make one concise commit message.

Also remove the * import, I'll put a comment on the line

File: server/monitor/src/main/java/org/apache/accumulo/monitor/rest/problems/ProblemsResource.java [hunk 1]
```diff
@@ -16,8 +16,7 @@
  */
 package org.apache.accumulo.monitor.rest.problems;
 
-import static org.apache.accumulo.monitor.util.ParameterValidator.ALPHA_NUM_REGEX;
-import static org.apache.accumulo.monitor.util.ParameterValidator.RESOURCE_REGEX;
+import static org.apache.accumulo.monitor.util.ParameterValidator.*;
```
Comment (mjwall) [inline server/monitor/src/main/java/org/apache/accumulo/monitor/rest/problems/ProblemsResource.java]: Remove the * import

**Comment by jzgithub1** [comment]
I got on it.  I will repush

**Comment by jzgithub1** [comment]
Performed a rebase that removed some UI commits from my accumulo-927 branch.  Removed those UI mods and only kept the ones related to this fix for #918.

**Pull request merged**